### ISSUE TITLE
Retro fit OS "adapt_framework: Marking aria-label does not read" fix

### ISSUE THREAD

**Issue by deltanetdan** [deltanet/adapt-contrib-core#3]

**Comment by deltanetdan** [comment]
See [issue/3](https://github.com/deltanet/adapt-contrib-core/tree/issue/3)
